src/main/java/fi/aalto/cs/apluscourses/ui/courseproject/CourseProjectActionDialogs.java

```diff
@@ -1,9 +1,9 @@
-package fi.aalto.cs.apluscourses.ui.base;
+package fi.aalto.cs.apluscourses.ui.courseproject;
 
+import com.intellij.openapi.project.Project;
 import org.jetbrains.annotations.NotNull;
 
-public interface Dialogs {
-  void showInformationDialog(@NotNull String message, @NotNull String title);
+public interface CourseProjectActionDialogs {
 
   void showErrorDialog(@NotNull String message, @NotNull String title);
```

Comment:
Nit: please consider unifying the API by making all the methods returning say `boolean`

Reply:
`false`? :roll_eyes:

Reply:
I understand it as a representation of the state of the `showErrorDialog`